help/de/Help_ch_GUIMenus.xml

```diff
@@ -3264,7 +3267,7 @@
       <tip>
         <para>Das Aussehen der Anzeige eines Kontobuchs ist in hohem Maße konfigurierbar. (siehe
           <ulink
-url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.html#txns-regstyle1">
+url="https://www.gnucash.org/docs/v&series-stable;/de/gnucash-guide/chapter_txns.html#txns-regstyle1">
```

Comment:
IMHO we should use this entities instead
<!ENTITY url-docs-build-C "https://code.gnucash.org/docs/C/gnucash-">
<!ENTITY url-docs-build-de "https://code.gnucash.org/docs/de/gnucash-"> …

Can you apply the change on files in all languages?